index.html

```diff
@@ -4,6 +4,7 @@
     <title>
         Kenny Suh's Blog
     </title>
+    <meta name=”viewport” content=”width=device-width, initial-scale=1”>
```

Comment:
This has a subtle bug, but it causes a pretty big difference to how your page displays. Essentially, this line isn't doing anything. If you look at your page on a mobile device, or an emulator, you should notice that your page isn't rendering responsively even though this line should make that happen.

What you have here is Smart Quotes `“` (Unicode U+201C), but what you need is single or double quotes (`'` or `"`). Can you see how the first ones are slightly angled? You actually probably don't even have 
This usually happens when you copy and past code from a blog or something that renders the text differently than how it really should be typed. It's super hard to notice if you're not looking for it. 

Also, once you use proper quotes, this still isn't quite right. You want the content attribute to be only `content="width=device-width"`

So this whole line should be:
```html
<meta name="viewport" content="width=device-width", initial-scale=1>
```